### DISCUSSION TITLE
ESP32-CAM, TFT and Internal Card

### DISCUSSION THREAD

**Discussion by nerkdesign** [Bodmer/TFT_eSPI#2766] [General]
Hello,

I did an Arduino code to take a picture on internal SD of an ESP32-CAM, it works.
I did another code to show the camera flow using your librairies TFT_eSPI and TJpeg_Decoder in a ILI9341 scren, it works.

But I can figure out to join these two codes.

I did multiple research on Web, but I'm not sure that it's possible. Can we use a ILI9341 screen and the internal SD card at the same time (if yes what pin should be use and how ?)?

Many thanks for your help and for your librairies.

**Comment by Bodmer** [comment]
Have you tried this example?
https://github.com/Bodmer/TJpg_Decoder/blob/master/examples/SD%20Card/SD_Jpg/SD_Jpg.ino

**Comment by nerkdesign** [comment]
Hi, 

Thanks for the reply, I indeed tried this example but still both not working maybe you have a pin map associated to this code ? I think this is just linked to my TFT and SD pin defined in my User_setup :/ thanks again for your help
